roles/activemq/tasks/main.yml

```diff
@@ -100,8 +100,10 @@
       no_log: true
 
     - name: Change location of ActiveMQ logs
+      vars:
+        activemq_log_config: "{{ 'log4j2' if activemq_version >= '5.17.0' else 'log4j' }}"
```

Comment:
Too bad you're still not using the `version` filter mentioned by @gionn . You could end up with unexpected results.
Check the examples https://docs.ansible.com/ansible/latest/collections/ansible/builtin/version_test.html#examples

Reply:
True, I missed that one.